bin/util

```diff
@@ -20,13 +20,19 @@ function announce() {
 }
 
 function ensure_client_is_generated(){
-    client_lang=$1
+    client_lang="$1"
+    version="$2"
     if [ $client_lang = "" ]; then
         echo "ensure_client_is_generated requires the client language as an argument"
         exit 1;
     fi
 
-    if [ ! -d "./out/$client_lang" ]; then
+    if [ $version = "" ]; then
+        echo "You must specify either dap or conjur as the version"
```

Comment:
```suggestion
        echo "You must specify either enterprise or conjur as the version"
```
could we use `edition` instead of `version` here?

Reply:
version makes me think of 1.1.2

Conjur is one product, with multiple editions